### ISSUE TITLE
Tiny Markdown syntax mistake in CHANGELOG.md

### ISSUE THREAD

**Issue by merkys** [aiidateam/aiida_core#1921]
From [v0.12.2/CHANGELOG.md](../blob/v0.12.2/CHANGELOG.md):
```
### Miscellaneous
- Remove pycrypto dependency, as it was found to have sercurity flaws [[#1754]](https://github.com/aiidateam/aiida_core/pull/1754)
- Set xsf as default format for structures visualization [[#1756]](https//github.com/aiidateam/aiida_core/pull/1756)
- Delete unused `utils/create_requirements.py` file [[#1702]]](https//github.com/aiidateam/aiida_core/pull/1702)
```
Last line has a syntax mistake.

**Comment by sphuber** [comment]
Thanks @merkys Given that the release branch is already merged and released, I will fix this in the merge of `master` back into `develop`

**Comment by merkys** [comment]
Great!

**Comment by sphuber** [comment]
Fixes in PR #1920
